Dockerfile

```diff
@@ -32,23 +32,11 @@
 # the case. Therefore, you don't have to disable it anymore.
 #
 
-FROM buildpack-deps:stretch AS base
+FROM golang:1.9.4 AS base
```

Comment:
Wondering if we should use:

```Dockerfile
ARG GO_VERSION=1.9.3
FROM golang:$GO_VERSION
```

Which would also allow testing on different versions if we'd want to

Reply:
Actually this means we can't test go rc's

Reply:
Should work, Docker Hub also pushes images for rc's

Reply:
@cpuguy83 to test RC's we could also "temporarly" refer to other images we build too, couldn't we ? 👼

Reply:
You can't do anything out of the norm! ;)

Actually didn't realize there were rc's on hub. That's great.

Reply:
We have a long tradition of pushing the RCs too: :innocent: 
```console
$ docker-hub-list-tags.sh library/golang | grep rc | grep -vE 'onbuild|wheezy|windowsservercore|nanoserver|cross'
rc
1.10-rc
1.10rc2
rc-stretch
1.10-rc-stretch
1.10rc2-stretch
rc-alpine
1.10-rc-alpine
1.10rc2-alpine
rc-alpine3.7
1.10-rc-alpine3.7
1.10rc2-alpine3.7
1.10rc1
1.10rc1-alpine
1.10rc1-alpine3.7
1.10rc1-stretch
1.9-rc-alpine
1.9rc2-alpine
rc-alpine3.6
1.9-rc-alpine3.6
1.9rc2-alpine3.6
1.9-rc
1.9rc2
1.9-rc-stretch
1.9rc2-stretch
1.9rc1-alpine
1.9rc1-alpine3.6
1.9rc1
1.9rc1-stretch
1.8-rc-alpine
1.8rc3-alpine
1.8-rc-stretch
1.8rc3-stretch
1.8-rc
1.8rc3
1.8rc2-alpine
1.8rc2
1.8rc1
1.8rc1-alpine
1.7rc6-alpine
1.7rc6
1.7rc5-alpine
1.7rc5
1.7rc4-alpine
1.7rc4
1.7rc3
1.7rc3-alpine
1.7rc2-alpine
1.7rc2
1.7rc1-alpine
1.7rc1
1.6rc2-alpine
1.6rc2
1.6rc1-alpine
1.6rc1
1.5rc1
1.4rc2
1.4rc1
```
(back to Go 1.4rc1 -- https://github.com/docker-library/golang/pull/26; 2014! :smile:)